Makefile

```diff
@@ -309,19 +309,29 @@ tests/interactive/gas-analysis/%.evm.test: tests/interactive/gas-analysis/%.evm
 
 # ProofTests
 
-proof_dir:=tests/proofs/specs
-proof_tests=$(wildcard $(proof_dir)/*/*-spec.k)
+proof_repo:=tests/proofs
+proof_specs_dir:=$(proof_repo)/specs
+proof_tests=$(wildcard $(proof_specs_dir)/*/*-spec.k)
 
 test-proof: $(proof_tests:=.test)
 
 test-java: tests/ethereum-tests/BlockchainTests/GeneralStateTests/stExample/add11_d0g0v0.json
 	./kevm run --backend java $< | diff - tests/templates/output-success-java.json
 
-$(proof_dir)/%.test: $(proof_dir)/% split-proof-tests
+$(proof_specs_dir)/%.test: $(proof_specs_dir)/% split-proof-tests
 	$(TEST) test --backend $(TEST_SYMBOLIC_BACKEND) $<
 
-split-proof-tests: tests/proofs/make.timestamp
-	$(MAKE) -C tests/proofs $@
+split-proof-tests: $(proof_repo)/git-submodule
+	$(MAKE) --directory tests/proofs/resources      $@
+	$(MAKE) --directory tests/proofs/bihu           $@
```

Comment:
you would need to drop bihu

Reply:
It passes, so we decided offline to keep it until it doesn't pass (so that we're notified of the change).